### PR TITLE
Demo submission: Kubeflow on AWS

### PR DESCRIPTION
**Pull request by cpiehl1**
Added demo and repo link to readme

### CODE REVIEW

**Comment by George-Bassilious** [comment]
What a nice demo! I really enjoyed the proffesionalism and the feeling I got after watching the demo. Here are some pointers I got after watching:
 
## Positives
*  The background musik makes the demo more immersive
* very good background content in the beginning, I felt that i Really got a good picture of the background information
* Good demonstration all in all, I felt that it was intuitive, entertaining and very well done!

## Possible suggestions for improvement

* The background musik felt to be a little loud sometimes which made it more diffuicult to hear the voice over in some parts. eg: 1:17
* There were no subtitles which could be helpful to have to really understand the voice over, Youtube autogenerates them so it was not a big issue however sometimes it is incorrect and a tip would be to add your own subtitles to the video
* sometimes it felt very cluttery on the screen and it was difficult to read the command lines, perhaps you can add subtitles or textboxes to show the commands in a bigger way

**Comment by mans-andersson** [comment]
# Demo day feedback
## Overall impressions
The voiceover is very clear and well made. It is
nice that you have a section which clearly motivates
why the topic matters to DevOps. The tempo of the actual
tutorial is very high and it is a bit tricky to follow
along with everything that happens. This is of course
difficult to avoid due to the 3 minute limit but 
you do have 9 more seconds to add if you'd like. 
Overall the video feels professional, good job! Down
below follows some actionable points you could change
to make the video better.

## Music
I liked the music but I feel that the music volume should
be lowered a little bit more to make the voiceover stand 
out better.

## Subtitles
The subtitles that are available are the auto-generated 
Youtube subtitles. They work very well for the most part
(impressive algorithm) but it does make some mistakes.
For example it translates Kubeflow to cube flow. You could
use the auto-generated subtitles as a starting point to
make 100% correct subtitles.

**Comment by annsudo** [comment]
Good tutorial, lots of work and time invested.

The content: clear and easy to follow, interesting topic, video include motivation on why the topic matter for DevOps
The video is appealing. Colourful graphics, background music, clear voice over with just-right tempo and calm temper (not easy when you given 3 min)

Suggestions for improvement: 
* the loud music, in the beginning, makes it difficult to hear the voice over
* the size of the text is small and is hard to follow, suggestion is to write the comments in the description of the video (in that way you would not need to re-edit the video)

**Comment by jhammarstedt** [comment]
# Feedback for demo #1501 

## Positive

* Good motivation of why this is relevant
*  Great explanation of kubeflow
* Overall visually appealing with good illustrations 

## Suggested improvements

* Background music was a bit too high a times, quick fix to just lower it a little 
* Lack of subtitles

Overall I think the video was nice and you demonstrated Kubeflow in a organised way.
Sometimes it was hard to see ad follow all the code, but given that this was a demo with 3 min deadline it's understandable.  Great job! 😊
